src/configs/chain_config.mainnet.json

```diff
@@ -17,8 +17,8 @@
   "primaryTokenUnit": "udsm",
   "votingPowerTokenUnit": "udsm",
   "tokenUnits": {
-    "udsm": {
-      "display": "dsm",
+    "xprt": {
+      "display": "xprt",
       "exponent": 6
```

Comment:
Please take a look of this document: https://forbole.atlassian.net/wiki/spaces/BD/pages/1769533/Create+new+explorer
And change line.17-21 accordingly.